plots/plot_curtailment.py

```diff
@@ -0,0 +1,132 @@
+import os
+import sys
+sys.path.append("../submodules/pypsa-eur")
+import pypsa
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import cartopy.crs as ccrs
+import logging
+import colors as c
+import warnings
+warnings.filterwarnings("ignore")
+from _helpers import mock_snakemake, update_config_from_wildcards, load_network, \
+                     change_path_to_pypsa_eur, change_path_to_base
+
+logger = logging.getLogger(__name__)
+
+RESULTS_DIR = "plots/results"
+
+
+def get_curtailment(n, nice_name):
+    curtailments = n.statistics()[["Curtailment"]]
+    techs = {"Solar": ["solar rooftop", "Solar"],
+             "Wind": ["Offshore Wind (AC)", "Offshore Wind (DC)", "Onshore Wind"]
+             }
```

Comment:
```suggestion
    techs = {
             "Solar Rooftop PV": ["solar rooftop"],
             "Solar Utility PV": ["Solar"],
             "Onshore Wind": ["Onshore Wind"],
             "Offshore Wind": ["Offshore Wind (AC)", "Offshore Wind (DC)"]
             }
```

I wouldn't aggregate techs other than Offwind, because there's a lot of interest in off-grid solutions (solar rooftop PV) as well as on to offshore ...